examples/service-catalog/OWNERS

```diff
@@ -1,6 +1,8 @@
 reviewers:
   - bparees
   - pmorie
+  - jboyd01
 approvers:
   - bparees
   - pmorie
+  - jboyd01
```

Comment:
i think it's up to @jwmatthews (or maybe @pmorie ?) whether you should have approval or not.  review seems fine.